resources/aws/asg_stack.go

```diff
@@ -142,16 +167,19 @@ func (s *ASGStack) CreateOrFail() error {
 
 // Update updates the autoscaling group stack in Cloud Formation if one of the
 // updatable parameters has changed.
-func (s *ASGStack) Update() error {
+func (s *ASGStack) Update() (bool, error) {
```

Comment:
This is removed now and simplified the error handling a lot :) The logging will still show if there was an error creating / updating the CF stack.